docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md

```diff
@@ -0,0 +1,7 @@
+# Operation record
+
+This topic will describe the operation record in Nebula Dashboard.
```

Comment:
the refers to specified nouns. the operation record -->operation records

Reply:
[describe the operation record] sounds weird. -->This topic shows how to use the operation record feature in xxx.